### ISSUE TITLE
Locating and applying the `en@truecase` option isn't intuitive.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [darktable-org/darktable#17411]
**Is your feature request related to a problem? Please describe.**

1.	As https://github.com/darktable-org/darktable/issues/2078#issuecomment-2322921369 states:

	> Reading entirely lower-case labels is difficult for me. You can see in the responses to https://ux.stackexchange.com/q/33708/132515 that the data is inconclusive on this topic (although [this revision of this answer](https://ux.stackexchange.com/revisions/33744/1#:~:text=Your%20name%20is%20actually%20the,feels%20more%20French%20to%20me.) provides a useful explanation of my opinion on this).

1.	As https://github.com/darktable-org/darktable/issues/2078#issuecomment-2315695311 states:

	> I had to search within the closed issues section on this tracker to ascertain that the option would be listed as a translation, and even then, the fact that a restart of the application is necessary isn't communicated to the user - I expected that the feature didn't function after I hit apply (because there was no success feedback there either).

Summarily, the sentence case feature isn't easily discoverable - it's in the language flyout, despite not being a language - and applying it isn't intuitive:

1.	The "Save" button doesn't indicate success; and:
1.	The "Restart to apply" pill notification appears solely in the parent window, despite the child configuration window not closing automatically upon application (which would be undesirable, but would at least display the notification to the user).

**Describe the solution you'd like**

As https://github.com/darktable-org/darktable/issues/2078#issuecomment-2323013164 states:

> > What is your proposal?
> 
> My best idea is to have a checkbox appear when `en-.*` is selected, which is labelled "Sentence case".

I can't imagine that having a checkbox appear or disappear based upon the chosen translation would be problematic to implement, and it would prevent the translation flyout being used to non-translation language stylisation attributes.

**Alternatives**

Most assistance is provided in tooltips:

> ![Screenshot_20240901_115200](https://github.com/user-attachments/assets/5aeb79bc-e95c-4798-971d-1538e3e48958)

However, as https://github.com/darktable-org/darktable/issues/2078#issuecomment-2323277612 states:

> Tooltips aren't inherently discoverable (due to there being no indication of whether a tooltip exists for an element) and don't function on touchscreen displays.
> 
> These are reasons that that KDE is moving to using contextual help buttons, per https://invent.kde.org/frameworks/kwidgetsaddons/-/merge_requests/240 and https://discuss.kde.org/t/element-specific-help-should-be-displayed-consistently/15624/2?u=rokejulianlockhart as others are.

Summarily, provide a button that invokes the tooltip, rather than a hover action, so that the user:

1.	Knows that an assistance tooltip is available; and:
1.	Knows that they are able to invoke it irrespective of their preferred input method.

The undermentioned is a step in the correct direction:

> ![image](https://github.com/user-attachments/assets/67f9f748-a723-4264-a92b-9c1618ebbbbc)

**Additional context**

Posted here per https://github.com/darktable-org/darktable/issues/2078#issuecomment-2323027798:

> The place for discussions is in open issues. If you have an idea to improve the UI please raise a new issue and discuss it there.

**Comment by elstoc** [comment]
> the sentence case feature isn't easily discoverable

You could propose an insert into the user manual to flag this up.

> Summarily, provide a button that invokes the tooltip, rather than a hover action

We should be consistent so this would be a change to the whole application if desired (I'm not sure it's a good idea). This would therefore be better as a separate feature request.

> I can't imagine that having a checkbox appear or disappear based upon the chosen translation would be problematic to implement

It's problematic if the translation gets dropped due to lapse in its maintenance, as it would lead to a non-functioning tick box. The whole idea of this change was to limit its impact by using pre-existing functionality.

**Comment by RokeJulianLockhart** [comment]
> We should be consistent so this would be a change to the whole application if desired (I'm not sure it's a good idea). This would therefore be better as a separate feature request.

@elstoc, I agree - it's rather outside the scope of this. I'd be glad to file it if you think it's of worth, or merely to have the issue linked here, if ever it's created.

> It's problematic if the translation gets dropped due to lapse in its maintenance, as it would lead to a non-functioning tick box. The whole idea of this change was to limit its impact by using pre-existing functionality.

Is that currently a concern?

> There's nothing stopping you from submitting a pull request to the repository proposing this change though.

@elstoc, surely you're aware that for the vast majority of people there is - I'm unable to develop in C, Lua, or C++. I'm still getting the ropes of damn CSS.

> One day I might get annoyed with all the American spellings and propose an en-GB translation, but without a sentence-case variant. In this case the proposed solution will not work

In which case, I could merely rescope this to `en-US`, surely?

**Comment by elstoc** [comment]
> Is that currently a concern?

With FOSS, people come and go. So I don't know if/when this will become a concern.

Anyway, as you might have gathered, apart from perhaps an addition to the user manual, I'm not sure I agree that these suggestions are necessary.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/darktable-org/darktable/issues/17411#issuecomment-2323288609

@elstoc, I've created https://github.com/darktable-org/dtdocs/pull/677#issue-2499406360 per your suggestion.

**Comment by elstoc** [comment]
> @elstoc, I've created https://github.com/darktable-org/dtdocs/pull/677#issue-2499406360 per your suggestion.

Thanks

**Comment by RokeJulianLockhart** [comment]
> https://github.com/darktable-org/darktable/issues/17411#issuecomment-2344434713

@victoryforce, because I've not been informed of a reason to. Surely that's inherent to such a statement? Apologies if I've misunderstood.

**Comment by beedell-roke** [comment]
> https://github.com/darktable-org/darktable/issues/17411#event-19548039438

Why on Earth would I just make a new issue with the same content, lacking all of this thread's context? That doesn't make any sense. Just reopen this. It's already been triaged.
